vevestaX/vevesta.py

```diff
@@ -17,7 +18,23 @@ def __init__(self ):
         self.variables = {}
                 
     def get_filename(self):
-        return ipynbname.name()
+        try:
+            try:
+                filename = ipynbname.name() + '.ipynb'
+            
+            except:
+                try:
+                    filePath = dict(inspect.getmembers(inspect.stack()[2][0]))['f_locals']['__file__']
+                except:
+                    filePath = sys.argv[0]
+                for i in range(len(filePath)-1,0,-1):
+                    if filePath[i] == "\\":
```

Comment:
why is it double slash..this might work only windows environment. Use os.base etc function to extract filename